internal/gocovshtest/testdata/errors/error_flows_missing_coverage_file_windows.golden

```diff
@@ -0,0 +1,11 @@
+                          
+ [1;38;2;255;85;85mCoverage report not found[0m
+                                                                           
+ Requested coverage profile is not found.                                  
+ By default, "coverage.out" is used. For other names, use "--profile" flag.
+                                                                                
+ [38;2;192;192;192mThe original error was:[0m                                                        
+ [38;2;192;192;192mopen testdata/general/missing.cover: The system cannot find the file specified.[0m
```

Comment:
I separated them because of the different error message windows has for file not found error. `The system cannot find the file specified` instead of `no such file or directory` like on linux and macos